### PR TITLE
Mb customer db

### LINKED ISSUE #13: Generate Customer Data
-use faker to create customer data objects
-requires unique_id, first_name, last_name, join_date

### PR DESCRIPTION
**Pull request by melissabell456**
1. Pull down this branch and run node build-js
2. Open DB Browser to review 20 employees added to customers table with first_name, last_name, customer_id,  join_date
3. issues resolved: #13  #15 
4. This should be merged with master because it builds customer faker data and adds it to the database locally

### CODE REVIEW

File: data/customers.json [hunk 1]
```diff
@@ -0,0 +1 @@
...eate_date":"2010-08-02"},{"first_name":"Harrison","last_name":"Emmerich","create_date":"2014-03-13"},{"first_name":"Rebecca","last_name":"Fadel","create_date":"2013-08-20"},{"first_name":"Vada","last_name":"Fritsch","create_date":"2017-12-11"},{"first_name":"Morgan","last_name":"MacGyver","create_date":"2014-11-16"},{"first_name":"Marcelino","last_name":"Block","create_date":"2013-06-03"},{"first_name":"Clark","last_name":"Schamberger","create_date":"2017-01-11"},{"first_name":"Ulices","last_name":"Boyle","create_date":"2016-04-03"},{"first_name":"Orville","last_name":"Grady","create_date":"2011-06-22"},{"first_name":"Horacio","last_name":"Carter","create_date":"2009-08-20"},{"first_name":"Wilber","last_name":"Paucek","create_date":"2017-07-12"},{"first_name":"Raegan","last_name":"Wintheiser","create_date":"2008-09-26"},{"first_name":"Yoshiko","last_name":"Olson","create_date":"2011-09-22"},{"first_name":"Violette","last_name":"Conroy","create_date":"2017-07-21"},{"first_name":"Darren","last_name":"Barrows","create_date":"2010-02-06"},{"first_name":"Edgardo","last_name":"Durgan","create_date":"2009-08-07"},{"first_name":"Johnson","last_name":"Donnelly","create_date":"2010-08-25"}]}
```
Comment (e-x-wilson) [inline data/customers.json]: Could you add line breaks on this file for readability?

**Review by electriceeli — CHANGES_REQUESTED** [review]
Requested change for JSON formatting. Comment on file.

**Review by hunterphillips — COMMENTED** [review]
👍

**Review by electriceeli — APPROVED** [review]
Looks good 👍

**Pull request merged**